.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Chromatic'
+
+on: push
+
+jobs:
+  chromatic:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Install dependencies
+        run: npm ci --maxsockets 1
+
+      - name: Build
+        run: npm run build -- --filter=@medplum/react
+
+      - name: Publish @medplum/react Chromatic
```

Comment:
Would it make more sense to include this in `build.yml` to avoid duplicate `npm ci` and `npm run build`?